vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md

```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
 
 The value of each cell in the map is gray at first, and it changes from blue to red as the number of valid data in the cell accumulates. It is preferable to continue the calibration until each cell of the map becomes close to red. In particular, the performance near the stop depends strongly on the velocity of 0 ~ 6m/s range and the pedal value of +0.2 ~ -0.4, range so it is desirable to focus on those areas.
 
-### Check the recommended notifications for map updates
+### Diagnostics
+
+`accel brake map_calibrator` publishes diagnostics depending on calibration status.
```

Comment:
The `accel brake map_calibrator` publishes diagnostics message depending on the calibration status.